src/main/kotlin/Cv.kt

```diff
@@ -187,7 +190,7 @@ private fun serve(command: Command) {
 
     val url = urlParameter(command)
     val urlString = url.toString()
-    val serverSettings = HttpServerSettings(zip = true)
+    val serverSettings = HttpServerSettings(InetAddress.getByAddress(byteArrayOf(0, 0, 0, 0)), zip = true)
```

Comment:
Is it feasible to make this configurable by a command line option? Binding to 0.0.0.0 by default could open the application to the outside without noticing that by the user.

Reply:
Yes good idea with the command line argument. I just thought on getting it running in a Docker container and exposed by Docker. that didn't worked with localhost. I will try it to refactor to command parameter.